R/rsa.r

```diff
@@ -0,0 +1,16 @@
+#' RSA SHA1
+#'
+#' As described in \url{http://tools.ietf.org/html/rfc3447}.
+#'
+#' @param key secret key
+#' @param string data to securely hash
+#' @keywords internal
+#' @export
+rsa_sha1 <- function(key, string) {
```

Comment:
Sounds good. Right now `hmac_sha1` is in its own file `hmac.r`. Should I move both to `oauth-signature.r`, or move both to a new common `sha1.r` file?